biojava-alignment/src/main/java/org/biojava/nbio/alignment/SimpleGapPenalty.java

```diff
@@ -30,9 +32,13 @@
  *
  * @author Mark Chapman
  */
-public class SimpleGapPenalty implements GapPenalty {
+public class SimpleGapPenalty implements GapPenalty, Serializable {
```

Comment:
I saw the commit messages "Make classes serializable for Spark" earlier and reached out to see if might help (I currently work for the UC Berkeley AMPLab on genomics applications for Apache Spark).

Making these classes serializable in this way is not only unsafe, and may cause future problems with API compatibility, it is not necessarily the most efficient way to use Biojava classes in Spark.  A lot of static and non-threadsafe state including matrices and caches will also be serialized, leading to significant overhead and potential concurrency problems.